packages/webapp/src/containers/Finances/saga.js

```diff
@@ -349,6 +415,9 @@ export default function* financeSaga() {
   yield takeLeading(ADD_OR_UPDATE_SALE, addSale);
   yield takeLatest(GET_EXPENSE, getExpenseSaga);
   yield takeLatest(GET_FARM_EXPENSE_TYPE, getFarmExpenseTypeSaga);
+  yield takeLeading(ADD_CUSTOM_EXPENSE_TYPE, addCustomExpenseTypeSaga);
+  yield takeLeading(UPDATE_CUSTOM_EXPENSE_TYPE, updateCustomExpenseTypeSaga);
+  yield takeLeading(RETIRE_CUSTOM_EXPENSE_TYPE, retireCustomExpenseTypeSaga);
```

Comment:
Just FYI, when a reducer is not necessary for the action, you could create an action in this file:
ex. `export const addCustomExpenseType = createAction('addCustomExpenseTypeSaga');`
then `yield takeLeading(addCustomExpenseType,type, addCustomExpenseTypeSaga);`
and use the action in the component. So you would not have to create constants and actions in the actions file. (I believe so...)

Reply:
Thanks yes I get confused between what is new style and old style. I thought I had to create an action this way to be consistent. I can fix it.